docs/clients/dotnet/21.2/migration-to-gRPC.md

```diff
@@ -0,0 +1,630 @@
+# Migration to gRPC client
+
+TCP client is considered legacy. We recommend migrating to the gRPC client. 
+
+We decided to use gRPC as our communication protocol, as it enabled us to standardise client communication. Thanks to that, we benefit from the built-in gRPC functionalities like streaming, back-pressure etc., instead of building the custom implementations. 
+
+As gRPC is a multi-platform and widely adopted standard, it enabled us to provide a unified approach and deliver clients for other dev environments like Go, JVM, NodeJs, Rust.
+
+TCP client will still be getting the necessary bug fixes, but new database features will be only delivered for the gRPC client (e.g. persistent subscription to `$all`).
+
+Having all of that, we encourage you to migrate to the gRPC client. This document outlines the needed steps. Check also the [gRPC documentation](/clients/grpc/) for more details on how to use it.
+
+## Update the target Framework
+
+gRPC client doesn't support .NET Standard. The change comes from using different gRPC implementations for the specific .NET Framework version to tune the performance.
+
+If you were using it in the .NET Standard library, you have to update it to one of:
+- .NET 6 (`net6.0`),
+- .NET 5 (`net5.0`),
+- .NET Core 3.1 (`netcoreapp3.1`),
+- .NET Framework 4.8 (`net48`)
```

Comment:
well .net 48.....

Reply:
Fair point 😉 Removed in https://github.com/EventStore/documentation/pull/486/commits/bfc112498e63d9366d62fc5a7519150f5c0a9a21.